pkg/pb/bbgo.proto

```diff
@@ -0,0 +1,207 @@
+syntax = "proto3";
+
+package pb;
+
+option go_package = "../pb";
+
+service BBGO {
+    // should support streaming
+    rpc Subcribe(SubscribeRequest) returns (stream SubscribeResponse) {}
+
+    // request-response
+    rpc SubmitOrder(SubmitOrderRequest) returns (SubmitOrderResponse) {}
+    rpc CancelOrder(CancelOrderRequest) returns (CancelOrderResponse) {}
+    rpc QueryOrder(QueryOrderRequest) returns (QueryOrderResponse) {}
+    rpc QueryOrders(QueryOrdersRequest) returns (QueryOrdersResponse) {}
+    rpc QueryTrades(QueryTradesRequest) returns (QueryTradesResponse) {}
+    rpc QueryKLines(QueryKLinesRequest) returns (QueryKLinesResponse) {}
+}
+
+message SubscribeRequest {
+    repeated Subscription subscriptions = 1;
+}
+
+message Subscription {
+    string exchange = 1;
+    string channel = 2;  // book, trade, ticker
+    string symbol = 3;
+    int64 depth = 4;
+}
+
+message SubscribeResponse {
+    repeated SuccessResponse success_responses = 1;
+}
+
+message SuccessResponse {
```

Comment:
name it SubscribeResponse? because we can only choose one message type for response